index.js

```diff
@@ -29,13 +29,18 @@ const proxy = new Proxy({
 
 proxy.on('incoming', (data, meta, toClient, toServer) => {
   if (meta.name === 'custom_payload') {
-    if (data.channel === 'badlion:mods') return
+    if (data.channel === 'badlion:mods' || data.channel === 'FML|HS' || data.channel === 'FML') return
```

Comment:
should we include all the aliases https://i.imgur.com/J5ROPYa.png

Reply:
if you intend to support 1.13+
that is used in FML2